travisci/sql-unittest/test_dbs_schema_compliance.t

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#      http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Test::More;
+
+use Bio::EnsEMBL::Compara::Utils::Test;
+
+
+## Check that the schemas test databases are fully compliant with the SQL standard
+
+my $compara_dir = Bio::EnsEMBL::Compara::Utils::Test::get_repository_root();
+my $multitestdb = Bio::EnsEMBL::Compara::Utils::Test::create_multitestdb();
+my $db_name = $multitestdb->create_db_name('schema');
+
+my $db_dir = "${compara_dir}/modules/t/test-genome-DBs";
+foreach my $test_file_name (glob "${db_dir}/*/*/table.sql") {
+  my $short_name = $test_file_name;
+  $short_name =~ s{${db_dir}/}{};
+  my $statements = Bio::EnsEMBL::Compara::Utils::Test::read_sqls($test_file_name);
+  foreach my $server_mode (qw(TRADITIONAL ANSI)) {
+    subtest "$short_name in $server_mode mode", sub {
+        Bio::EnsEMBL::Compara::Utils::Test::test_schema_compliance($multitestdb, $db_name, $statements, $server_mode);
+    };
+  }
+}
```

Comment:
```suggestion
foreach my $test_file_name (glob "${db_dir}/*/*/table.sql") {
    my $short_name = $test_file_name;
    $short_name =~ s{${db_dir}/}{};
    my $statements = Bio::EnsEMBL::Compara::Utils::Test::read_sqls($test_file_name);
    foreach my $server_mode (qw(TRADITIONAL ANSI)) {
        subtest "$short_name in $server_mode mode", sub {
            Bio::EnsEMBL::Compara::Utils::Test::test_schema_compliance($multitestdb, $db_name, $statements, $server_mode);
        };
    }
}
```

Reply:
GitHub doesn't show it because almost every line has changed, but this file is based on other `.t` that I have edited, trying to fit with the existing indentation
```
commit 181ce512fffe3ecd96416b1f7e1834611b1f5990
 travisci/sql-unittest/test_dbs_schema_compliance.t | 14 ++------------

commit 299cfdcd8c5c77b1eca85c70ffa246159b8d70dc
 travisci/sql-unittest/{schema_test_dbs.t => test_dbs_schema_compliance.t} | 0

commit 3855bb41b8258f26d39ce9730314a1609d3dcfac
 travisci/sql-unittest/schema_test_dbs.t | 95 +++--------------------------------------------------------------------------------------------

commit 911e09c82c6ca44cfa338440e4266777dd3e2c22
 travisci/sql-unittest/schema_test_dbs.t | 6 +++---

commit 318e85ddbb7ad1604a4ab41e19496f3e7c410e75
 travisci/{all-housekeeping => sql-unittest}/schema_test_dbs.t | 5 ++++-

commit b071afde29c86e96c1cb24f0b33aec8abb0026a5
 travisci/all-housekeeping/{schema.t => schema_test_dbs.t} | 28 ++++++++++++++++------------

commit 447825916031efa2934f1c78ed6af58e9ba761fe
 {modules/t => travisci/all-housekeeping}/schema.t | 5 +++--

commit e4f8abb354d30d77f479c15e4ed9c620ad80a125
 modules/t/schema.t | 137 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```